build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
+            "url": "https://dotnet.myget.org/F/roslyn/api/v2/package"
+        }
+    ],
+    "vsixPush": "https://dotnet.myget.org/F/roslyn/vsix/upload",
+    "channels": [
+        {
+            "name": "dev15.5",
+            "kind": "PerBuildPreRelease"
```

Comment:
How does this kind relate to the other kind in the NuGet section? When would they be different?